woocommerce/woocommerce
# Link cross-sells product image to product page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #42032.

When using the classic cart, the product images of the cross-sells products are linked to the corresponding product pages. When using the Cart block, the product images of the cross-sells products are not linked. This PR synchs the behaviour between the classic cart and the Cart block.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that one product has at least one cross-sells product, and that cross-sells product is on sale. See https://woo.com/document/related-products-up-sells-and-cross-sells/
2. Go to the frontend and add the edited product to the cart.
3. Go to the Cart block and verify that the product title of the cross-sells product links to the corresponding product page, and the "Sale" badge displays properly.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Link cross-sells product images to corresponding product pages.

#### Comment 

</details>
